anomaly_detection/bayesian_changepoints.md

```diff
@@ -1,3 +1,4 @@
+<script src="../dato/js/recview.js"></script>
```

Comment:
I don't suppose there is a way we can modify the theme (?) of GitBook to automatically include this file in `.md` files? Or are there some `md` files we want to deliberately exclude? Doing it this way seems repetitive and error-prone (new files are likely to forget to include it).